alembic-autogenerate.py

```diff
@@ -3,27 +3,28 @@
 #
 # How to use this script:
 # 1. Load the LSST environment and setup sdm_schemas and felis.
-#        source loadLSST.bash
+#        source loadLSST.bash or activate your eups setup
```

Comment:
Alternately, the following should work:

```
pip install lsst-felis testing.postgresql alembic sqlalchemy pyyaml
```

And `sdm_schemas` could be cloned with: `git clone https://github.com/lsst/sdm-schemas`

It should be unnecessary to have a full LSST stack environment just to run the migrations.

Reply:
Should be able to `pip install` sdm_schemas itself and use the python package resources to find the schemas.

Reply:
> Should be able to `pip install` sdm_schemas itself and use the python package resources to find the schemas.

That is on the todo list.